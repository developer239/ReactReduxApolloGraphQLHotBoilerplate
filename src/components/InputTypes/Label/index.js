import styled from 'styled-components'


const Label = styled.label`
  display: block;
  font-weight: bolder;
  margin-bottom: 20px;
  
  input {
    display: inline-block;
    margin-left: 15px;
  }
`

export default Label
