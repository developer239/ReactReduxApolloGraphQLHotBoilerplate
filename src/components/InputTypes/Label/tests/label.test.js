import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'
import Label from '../index'


describe('Label Component', () => {
  it('has correct default styles', () => {
    const wrapper = shallow(
      <Label>Test Label</Label>,
    )
    expect(wrapper).toMatchSnapshot()
  })
})
