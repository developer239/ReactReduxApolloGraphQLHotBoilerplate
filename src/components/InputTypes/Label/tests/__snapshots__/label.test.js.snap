// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component has correct default styles 1`] = `
.c0 {
  display: block;
  font-weight: bolder;
  padding: 0;
  margin-bottom: 20px;
}

.c0 input,
.c0 textarea {
  display: block;
}

<label
  className="c0"
>
  Test Label
</label>
`;
